core/src/main/java/com/scalar/db/storage/cosmos/ValueBinder.java

```diff
@@ -1,100 +1,68 @@
 package com.scalar.db.storage.cosmos;
 
-import com.scalar.db.io.BigIntValue;
-import com.scalar.db.io.BlobValue;
-import com.scalar.db.io.BooleanValue;
-import com.scalar.db.io.DoubleValue;
-import com.scalar.db.io.FloatValue;
-import com.scalar.db.io.IntValue;
-import com.scalar.db.io.TextValue;
-import com.scalar.db.io.ValueVisitor;
+import com.scalar.db.io.BigIntColumn;
+import com.scalar.db.io.BlobColumn;
+import com.scalar.db.io.BooleanColumn;
+import com.scalar.db.io.ColumnVisitor;
+import com.scalar.db.io.DoubleColumn;
+import com.scalar.db.io.FloatColumn;
+import com.scalar.db.io.IntColumn;
+import com.scalar.db.io.TextColumn;
 import java.util.Base64;
 import java.util.function.Consumer;
 import javax.annotation.concurrent.NotThreadSafe;
 
 /**
- * A visitor class to bind {@code Value}s to a {@link StringBuilder}
+ * A visitor class to bind {@code Value}s to a {@link StringBuilder}.
  *
  * @author Yuji Ito
  */
 @NotThreadSafe
-public final class ValueBinder implements ValueVisitor {
+public final class ValueBinder implements ColumnVisitor {
```

Comment:
Are these intended changes for this PR? (looks like coming from a previous PR or it's left unchanged in the previous PR?)

Reply:
Thank you for taking at this PR!

In the previous PR, I didn't remove all the usages of `ValueVisitor`, and I thought I was going to do that in another PR. But the changes were needed for this `isNull` and `isNotNull` conditions support, so I did it in this PR. I will remove the rest of the usages of `ValueVisitor` in another PR. Thanks.